src/scss/includes/panels/poi_panel.scss

```diff
@@ -400,12 +431,48 @@ $BLOCK_INDENT: 60px;
 
 @media (max-width: 640px) {
   .poi_panel {
-    max-height: calc(100vh - 200px);
+    max-height: calc(100vh - 100px);
```

Comment:
On chrome mobile, "100vh" includes the address bar, which makes the panel too high, and difficult to close.

<img src="https://user-images.githubusercontent.com/4726554/49010149-a7e26d00-f172-11e8-823e-5580f16b7b7b.png" width="400">

Reply:
@nlehuby what do we do ? is add 30px is sufficient ?